.storybook/static/core-theme.css

```diff
@@ -136,7 +136,7 @@
 --site-margins-mobile: 1.5rem;
 --site-max-width: 1040px;
 --text-max-width: 53rem;
---image-path: "../images";
+--image-path: '../images';
```

Comment:
Yeah actually you're right these aren't used anywhere.

Reply:
The Sass variable list that we include to create _layout.scss for each system doesn't exclude those variables from being created as CSS vars. Is that something we want to do? I'm on the fence because I think a few extra variables that could be referenced aren't a bad thing, but I guess there could be some confusion if modifying them doesn't do anything.

Reply:
An alternative would be to hardcode the path into multiple places in our code...which might not be as terrible as it sounds. We only seem to use it once in core and twice in one module in healthcare.

Reply:
Yeah this is the best option.. I'll do this.

Reply:
Actually storybook might need this variable to set path correctly for prod builds, I'll mess with it, probably another way to do it.

Reply:
Nope all good was able to remove it.